src/Exporter.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry;
+
+use OpenTelemetry\Tracing\Span;
+use OpenTelemetry\Tracing\Tracer;
+
```

Comment:
not necessarily for this PR, but:

Should we be adding docstrings to our classes?  I feel as if they are very helpful (and might be even more helpful for future committers): 

http://phpdocu.sourceforge.net/howto.php

Reply:
i'm not sure that is really needed right now, i think we will get back to this later, because this can be helpful for api generation.

Reply:
It can also be helpful for new maintainers of the project!